models/github_user.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+type GithubUser struct {
+	Id          int
```

Comment:
Another not written rule is to write acronyms in all letters upper case.
```suggestion
	ID          int
```